website/versioned_docs/version-0.71/text.md

```diff
@@ -448,6 +448,16 @@ Specifies the disabled state of the text view for testing purposes.
 
 ---
 
+### `dynamicTypeRamp` <div class="label ios">iOS</div>
```

Comment:
It looks like https://github.com/facebook/react-native/commit/11c8bf313717a8c46165d48caba5e70b9f160d05 landed after 0.71 was cut. So it will be present in 0.72.

Reply:
Duly noted.